lib/multihandle.h

```diff
@@ -134,6 +139,13 @@ struct Curl_multi {
                                     previous callback */
   bool in_callback;            /* true while executing a callback */
   long max_concurrent_streams; /* max concurrent streams client to support */
+
+#ifdef ENABLE_UNBLOCK
+  curl_socket_t unblock_read_socket; /* reader half of the socketpair()
+                                        used for unblock */
+  curl_socket_t unblock_write_socket; /* writer half of the socketpair()
+                                        used for unblock */
```

Comment:
If I would've written this, I would've made the sockets like this:

`curl_socket_t unblock_pair[2];` 

As they can be created easier that way, and they are also so tightly coupled so having them within the same array seems a good idea. I'm not insisting on that it must be done like this, I'm just saying...

Reply:
done